proxy/fyde_proxy_troubleshoot.md

```diff
@@ -1,5 +1,9 @@
 # Troubleshoot
 
+- The steps in this page are best performed from a Desktop client
+
+- Since most Mobile clients don't provide the required tools, we are developing features to help troubleshooting from this type of platforms
```

Comment:
@GMartinez-Sisti perhaps `Troubleshooting using the mobile app is currently in development.`?

Reply:
That's better. Going to change.